source/manuals/programming-languages/js.html.md.erb

```diff
@@ -110,14 +110,6 @@ Use soft tabs with a two space indent.
   sets the appropriate affordances. It also follows the conventions of the
   standard library.
 
-## CoffeeScript
-
-Don't use CoffeeScript.
-
-**Why:** It's an extra abstraction and introduces another
-language for developers to learn. Using JavaScript gives us guaranteed
-performance characteristics and more well known support paths.
-
 ## HTML class hooks
 
 When attaching JavaScript to the DOM use a `.js-` prefix for the HTML classes.
```

Comment:
With v3 of Frontend using the namespace '.govuk-js', is it sensible to also change the JavaScript prefix advice changed to use '.app-js-' or '.gem-js'?

Reply:
I'm trying to keep his pull request is limited to factual changes only, additions like that I think can be done as a separate pull request in the future.

Think it's important to merge smaller things rather than having very big pull requests open for long periods of time.

Reply:
Sure - make sense. But the flip side is that it's highly likely it won't happen until the next review is due.